### PR TITLE
Feature/positioning

### PR DESCRIPTION
**Pull request by Nerdy-Girl**

### CODE REVIEW

File: docs/positioning.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  'header',
+  'body',
+  'footer'
+) !default;
+```
+
+## Mixin size
+
+### Description
+_Bootstrap helper which helps _
```
Comment (ArekRado) [inline docs/positioning.md]: please add better description ;)

**Review by ArekRado — CHANGES_REQUESTED** [review]

**Review by ArekRado — APPROVED** [review]

File: docs/positioning.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+### Parameters
+- `$name` - string with name which should be included in ```$z-indexes``` (***required***)
+
+### Usage: 
+Assigned appropriate z-index for header
+
+```scss
+.exampleZClass {
+    @include z('header');
```
Comment (ArekRado) [inline docs/positioning.md]: z-index

**Review by ArekRado — CHANGES_REQUESTED** [review]

File: docs/positioning.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+### Parameters
+- `$width` - variable describing width (***required***)
+- `$height` - variable describing height (as default it's used ```$width```)
+
+
+## Mixin clearfix
+
+### Description
+_Bootstrap mixin which helps with wrapping containers_
```
Comment (JSerhii) [inline docs/positioning.md]: Bootstrap helper. Used on a container of which all the children are floating. If you would not use this, the container would not get any height (except in IE)

File: docs/positioning.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+.exampleZClass {
+    @include z('header');
+}
+```
+
+
+## Mixin pseudo
+
+### Description
+_Mixin which helps with positioning element with content, display and position properties._
```
Comment (JSerhii) [inline docs/positioning.md]: ...positioning speudo elements

File: docs/positioning.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+- `$pos` - variable which will be used as a value for ```position``` (as default it's used ```absolute```)
+- `$content` - variable which will be used as a value for ```content``` (as default it's used empty string)
+
+### Usage: 
+Assigned element with inline display and fixed position
+
+```scss
+.examplePseudoClass {
+    @include pseudo($display: inline, $pos: fixed);
+}
```
Comment (JSerhii) [inline docs/positioning.md]:
```scss
.exampleClass {
   &:before {
       @include pseudo(block, absolute);
       ...
    }
   &:after {
       @include pseudo();
       ...
   }
}
```

File: docs/positioning.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+### Parameters
+- `$args` - variable(s) which will be used as a value(s) for ```top right bottom left``` properties (as default it's used empty string)
+
+### Usage: 
+Assigned position fixed without additional properties
+
+```scss
+.exampleFixedClass {
+    @include fixed();
```
Comment (JSerhii) [inline docs/positioning.md]:
```scss
...
@include fixed(left 50% bottom 150px);
...
```

File: docs/positioning.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+### Parameters
+- `$args` - variable(s) which will be used as a value(s) for ```top right bottom left``` properties (as default it's used empty string)
+
+### Usage: 
+Assigned position absolute without additional properties
+
+```scss
+.exampleAbsoluteClass {
+    @include absolute();
```
Comment (JSerhii) [inline docs/positioning.md]:
```scss
...
@include absolute(top 0 left 0 bottom 0);
...
```

File: src/_positioning.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+@function z($name) {
+  @if index($z-indexes, $name) {
+    @return (length($z-indexes) - index($z-indexes, $name)) + 1;
+  } @else {
+    @warn 'There is no item "#{$name}" in this list; choose one of: #{$z-indexes}';
+    @return null;
+  }
+}
+
+@mixin pseudo($display: block, $pos: absolute, $content: '') {
```
Comment (JSerhii) [inline src/_positioning.scss]: I think we can remove prop `position` from this mixin and use mixins `absolute` or `relative` for pseudo elements

**Review by JSerhii — CHANGES_REQUESTED** [review]

**Review by JSerhii — APPROVED** [review]

**Review by ArekRado — APPROVED** [review]

**Pull request merged**